src/OmniSharp.LanguageServerProtocol/LanguageServerLoggerFactory.cs

```diff
@@ -33,5 +34,30 @@ public void Dispose()
         {
             throw new NotImplementedException();
         }
+
+        private static bool LogFilter(string category, LogLevel level, IOmniSharpEnvironment environment)
```

Comment:
I moved this here because LSP is the only one using this method now. It might be that this is unneded, but LSP's logging looked a bit fancier, so I left it.

Reply:
That should be fine, I'll see if I can take a look at lsp logging later.